CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(robot_localization)
 
+# FIXME(jservos): AVX possibly causing memory corruption in Eigen
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mno-avx")
```

Comment:
I'd do `add_extra_compiler_option(-mno-avx)` instead, as in https://github.com/clearpathrobotics/opencv3-release/blob/release/kinetic/opencv3/cmake/OpenCVCompilerOptions.cmake#L199

Reply:
Unknown CMake command "add_extra_compiler_option".